tests/functional/event_handler/_pydantic/test_openapi_params.py

```diff
@@ -388,6 +389,19 @@ def handler(user: Annotated[User, Body(description="This is a user")]):
     assert request_body.content[JSON_CONTENT_TYPE].schema_.description == "This is a user"
 
 
+def test_openapi_with_deprecated_operations():
```

Comment:
We could also test other operations (`post`, `delete`, etc) and routers (`Router`, etc).

Reply:
Since all the operations inherit from the same class, I don't think we need all of them. But please add at least a get and a post, just to make sure it work in different methods 😄